libsolidity/lsp/SemanticTokensBuilder.cpp

```diff
@@ -118,7 +118,7 @@ void SemanticTokensBuilder::encode(
 	auto const [line, startChar] = m_charStream->translatePositionToLineColumn(_sourceLocation.start);
 	auto const length = _sourceLocation.end - _sourceLocation.start;
 
-	lspDebug(fmt::format("encode [{}:{}..{}] {}", line, startChar, length, _tokenType));
+	lspDebug(fmt::format("encode [{}:{}..{}] {}", line, startChar, length, static_cast<int>(_tokenType)));
```

Comment:
Note in the old version fmtlib formats enum class as an integer but in the later version you need to provide a custom formatter. This preserves the original behavior.